omnigibson/robots/tiago.py

```diff
@@ -509,9 +509,18 @@ def gripper_control_idx(self):
     def finger_lengths(self):
         return {arm: 0.12 for arm in self.arm_names}
 
+    @property
+    def disabled_collision_link_names(self):
+        return [name for arm in self.arm_names for name in [f"arm_{arm}_tool_link", f"wrist_{arm}_ft_link", f"wrist_{arm}_ft_tool_link"]]
+
```

Comment:
Hmmm why are these suddenly needed?

Reply:
add comment -- these should NEVER have collisions anyways

Reply:
done